cmd/podman/inspect/inspect.go

```diff
@@ -192,15 +258,55 @@ func (i *inspector) inspectAll(ctx context.Context, namesOrIDs []string) ([]inte
 			data = append(data, ctrData[0])
 			continue
 		}
+		i.podOptions.NameOrID = name
+		podData, err := i.containerEngine.PodInspect(ctx, i.podOptions)
+		if err != nil {
+			if errors.Cause(err) == define.ErrNoSuchPod {
+				errs = []error{err}
+			} else {
+				return nil, nil, err
+			}
+		} else {
+			errs = nil
+			data = append(data, podData)
+			continue
+		}
 		imgData, errs, err := i.imageEngine.Inspect(ctx, []string{name}, i.options)
 		if err != nil {
```

Comment:
I suspect this is going to need a check for no-such-image

Reply:
`imageEngine.Inspect` puts no such image into errs, same as `ContainerInspect`. They work differently from `NetworkInspect` and `VolumeInspect`, which also work differently from PodInspect